pheps/phep-0001.md

```diff
@@ -0,0 +1,338 @@
+
+9. **Reference Implementation** - The reference implementation must be completed before any PHEP is given status "Final", but it need not be completed before the PHEP is accepted. While there is merit to the approach of reaching consensus on the specification and rationale before writing code, the principle of "rough consensus and running code" is still useful when it comes to resolving many discussions of API details.
+
+    The final implementation must include test code and documentation appropriate for the relevant PyHC project(s).
+
+10. **Rejected Ideas** - Throughout the discussion of a PHEP, various ideas will be proposed which are not incorporated. Those rejected ideas should be recorded along with the reasoning as to why they were rejected. This helps record the thought process behind the final version of the PHEP and prevents people from bringing up the same rejected idea again in subsequent discussions.
+
+11. **Open Issues** - While a PHEP is in draft, ideas can come up which warrant further discussion. Those ideas should be recorded so people know that they are being thought about but do not have a concrete resolution. This helps make sure all issues required for the PHEP to be ready for consideration are complete.
+
+12. **Footnotes** - A collection of footnotes cited in the PHEP, and a place to list non-inline hyperlink targets.
+
+13. **Copyright/license** - Each new PHEP must be placed under a dual license of public domain and [CC0-1.0-Universal](https://creativecommons.org/publicdomain/zero/1.0/) and include citation information (see this PHEP for an example).
+
+# PHEP Formats and Templates
+<a name="phep-formats-and-templates"></a>
+PHEPs are UTF-8 encoded text files using the Markdown format. A future PHEP may include a template. PHEP Markdown must be compatible with the most recent version of the [CommonMark specification](https://spec.commonmark.org/) at the time the PHEP is approved. PHEPs must not use [GitHub extensions](https://github.github.com/gfm/).
+
+The editors are responsible for enforcing PHEP format and consistency of style within a PHEP. They may use automated tools (e.g. pre-commit hooks) to assist.
+
+# PHEP Header Preamble
+<a name="phep-header-preamble"></a>
+Each PHEP must begin with an [RFC 2822](https://datatracker.ietf.org/doc/html/rfc2822.html) style header preamble, marked off with a [code fence](https://spec.commonmark.org/current/#fenced-code-blocks). The headers must appear in the following order. Headers marked with "*" are optional. All other headers are required.
+
+      ```
+      PHEP: <phep number>
+      Title: <phep title>
+      Author: <list of authors' real names and optionally, email addrs>
+      Discussions-To: <URL of current canonical discussion thread>
+      Status: <Draft | Accepted | Rejected |
+               Withdrawn | Final | Superseded>
```

Comment:
Author can include ORCiD, discussions-To is PR thread, status is Replaced not Superseded--will capture next push.